### DISCUSSION TITLE
Comparing Zustand with Jotai: access outside of React

### DISCUSSION THREAD

**Discussion by dmytro-ulianov** [pmndrs/zustand#2245] [Q&A]
In the [docs](https://docs.pmnd.rs/zustand/getting-started/comparison#state-model-(vs-jotai)) it is stated, that "_a Zustand store is an external store, making it more suitable when access outside of React is required_", and i want to double-check if it's still the case or this part needs a refreshment. 

Jotai v2 has a [Store API](https://jotai.org/docs/core/store#createstore), that is designed to give you the access to atoms outside of the react.

**Comment by dbritto-dev** [comment]
@dmytro-ulianov nope, same behavior. You can use zustand outside of React without any issue

**Reply by dmytro-ulianov** [reply]
Oh, now i see that i didn't ask any actual question. 
My question was regarding the comparison with Jotai. The docs states that Zustand can be used outside of react, suggesting that Jotai cannot. I would like to know if this statement still holds true, or if Jotai essentially has the same functionality as of version 2?
